starter-code/index.js

```diff
@@ -1,17 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items = this.items.sort((a, b) => a - b);
+  }
 
-  get(pos) {}
+  get(pos) {
+    let value = this.items[pos];
+    if (pos === this.items[pos]) {
+      return value;
+    } else {
+      throw new Error("OutOfBounds");
+    }
+  }
 
-  max() {}
+  max() {
+    let max = this.items.reduce((a, b) => {
```

Comment:
you used the Math.max. No need to iterate using the reduce.
```js
return Math.max(...this.items);
```
Let the javascript magic do the rest